redis.conf

```diff
@@ -0,0 +1,2 @@
+requirepass goodPassword
```

Comment:
the user will use this Redis configuration file to set up the Redis database for test